kvs_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestKVS(t *testing.T) {
+
+	t.Run("not found", func(t *testing.T) {
+		kvs := NewMyKVS()
+
+		_, err := kvs.Load("SjknVAAA")
+		if nil == err {
+			t.Fail()
+		}
+		if "not found" != err.Error() {
+			t.Fail()
+		}
+	})
+
+	t.Run("store", func(t *testing.T) {
+		kvs := NewMyKVS()
+
+		err := kvs.Store("SjknVAAA", "golang.org")
+		if nil != err {
+			t.Fail()
+		}
+	})
+
+	t.Run("already exist", func(t *testing.T) {
+		kvs := NewMyKVS()
+
+		kvs.Store("SjknVAAA", "golang.org")
+		err := kvs.Store("SjknVAAA", "golang.org")
+		if nil == err {
+			t.Fail()
+		}
+		if "already exist" != err.Error() {
```

Comment:
Testing for a specific error string by comparing the message returned by `err.Error()` is not advised since the error message could change. When you do want to test against a specific error and not an error _in general_, then it's preferable to test against the error type or error variable (sentinel errors).

Resources: more [here](https://blog.golang.org/go1.13-errors)

TLDR: One way to _solve_ this:
Adding a `var errAlreadyExists errors.New("can't add store entry: already exists")` global, returning it when and comparing against it in tests.